Capstone-Projects-2022-Spring/project-vicsr
# Vic 175 users can access document page without being logged in when hitting register button

Bug(s) fixed

Testing instructions:
1) checkout branch and npm start
2) try to log in with an invalid account
3) press "register" button and notice that it performs expected behavior
4) try to use an invalid login (either restarting localhost or not)
5) refresh page, notice it simply brings you back to the login page
